python/test/pyMaterial_TEST.py

```diff
@@ -0,0 +1,244 @@
+
+    self.assertEqual(Color(0, 0, 0, 1), material.diffuse())
+    material.set_diffuse(Color(0.2, 0.3, 0.4, 0.6))
+    self.assertEqual(Color(0.2, 0.3, 0.4, 0.6), material.diffuse())
+
+    self.assertEqual(Color(0, 0, 0, 1), material.specular())
+    material.set_specular(Color(0.3, 0.4, 0.5, 0.7))
+    self.assertEqual(Color(0.3, 0.4, 0.5, 0.7), material.specular())
+
+    self.assertEqual(Color(0, 0, 0, 1), material.emissive())
+    material.set_emissive(Color(0.4, 0.5, 0.6, 0.8))
+    self.assertEqual(Color(0.4, 0.5, 0.6, 0.8), material.emissive())
+
+    self.assertTrue(material.lighting())
+    material.set_lighting(False)
+    self.assertFalse(material.lighting())
+
+    self.assertEqual(0, material.render_order())
+    material.set_render_order(5)
+    self.assertEqual(5, material.render_order())
+
+    self.assertFalse(material.double_sided())
+    material.set_double_sided(True)
+    self.assertTrue(material.double_sided())
+
+    self.assertEqual("", material.script_uri())
+    material.set_script_uri("uri")
+    self.assertEqual("uri", material.script_uri())
+
+    self.assertEqual("", material.script_name())
+    material.set_script_name("name")
+    self.assertEqual("name", material.script_name())
+
+    self.assertEqual(Material.ShaderType.PIXEL, material.shader())
+    material.set_shader(Material.ShaderType.VERTEX)
+    self.assertEqual(Material.ShaderType.VERTEX, material.shader())
+
+    self.assertEqual("", material.normal_map())
+    material.set_normal_map("map")
+    self.assertEqual("map", material.normal_map())
+
+    self.assertEqual("", material.file_path())
+    material.set_file_path("/my/path")
+    self.assertEqual("/my/path", material.file_path())
+
+    # TODO(ahcorde) Add Pbr python interface
+    # set pbr material
+    # sdf::Pbr pbr
+    # sdf::PbrWorkflow workflow
+    # workflow.SetType(sdf::PbrWorkflowType::METAL)
+    # pbr.SetWorkflow(workflow.Type(), workflow)
+    # material.SetPbrMaterial(pbr)
+    # EXPECT_NE(material.PbrMaterial(), nullptr)
+    # self.assertEqual(workflow,
+    #   *material.PbrMaterial()->Workflow(sdf::PbrWorkflowType::METAL))
+
+    # Move the material
+    moved = material
+    self.assertEqual(Color(0.1, 0.2, 0.3, 0.5), moved.ambient())
+    self.assertEqual(Color(0.2, 0.3, 0.4, 0.6), moved.diffuse())
+    self.assertEqual(Color(0.3, 0.4, 0.5, 0.7), moved.specular())
+    self.assertEqual(Color(0.4, 0.5, 0.6, 0.8), moved.emissive())
+    self.assertFalse(moved.lighting())
+    self.assertEqual(5, moved.render_order())
+    self.assertTrue(moved.double_sided())
+    self.assertEqual("uri", moved.script_uri())
+    self.assertEqual("name", moved.script_name())
+    self.assertEqual(Material.ShaderType.VERTEX, moved.shader())
+    self.assertEqual("map", moved.normal_map())
+    # TODO(ahcorde) Add Pbr python interface
+    # self.assertEqual(workflow,
+    #   *moved.PbrMaterial()->Workflow(sdf::PbrWorkflowType::METAL))
+    self.assertEqual("/my/path", moved.file_path())
```

Comment:
These checks can be removed since there no move assignment in python.

Reply:
https://github.com/ignitionrobotics/sdformat/pull/948/commits/7b42ea9b03e17d2be4d5979326f0322af659961d

Also added a test for deepcopy in https://github.com/ignitionrobotics/sdformat/pull/948/commits/d9a68bf6a41afee265ebb1d9bfc762352312bd45